tools/sof-kernel-log-check.sh

```diff
@@ -211,6 +211,9 @@ ignore_str="$ignore_str"'|usb .-.+: device descriptor read/.+, error'
 ignore_str="$ignore_str"'|usb .-.+: device not accepting address .+, error'
 ignore_str="$ignore_str"'|usb usb.-port.+: unable to enumerate USB device'
 
+# usb 3-8-port4: disabled by hub (EMI?), re-enabling...
```

Comment:
Is this happened only once?
We need some more basic information in comment block to track later also.

```
# found on TGLU_SKU0A32_SDCA with check-playback-100sec.sh, internal daily test #5745
# see also: https://github.com/thesofproject/linux/blob/b73297355e03ffdff62e3d7a6438934f05c58f54/drivers/usb/core/hub.c#L5563
```

Reply:
Thanks for your advice. I find this error happened only once in our tests, so maybe it was just caused by EMI coupling.